_plugins/product_versions.rb

```diff
@@ -0,0 +1,34 @@
+module ProductVersion
```

Comment:
I think we are actively trying to avoid plugins because it breaks Github Pages compatibility. @johndmulhausen ?